src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+// import React from 'react';
+import React from 'react';
+import Calculator from './components/Calculator';
+import './components/Calculator.css';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
```

Comment:
Great design @ijeomaodinko 🤝

- [ ] [Optional] Kindly use ES6 syntax for your functions. This is the recommended practice for all Javascript projects